ansible/idr-centos-init.yml

```diff
@@ -4,7 +4,16 @@
 - hosts: "{{ idr_environment | default('idr') }}-hosts"
 
   tasks:
-  - name: Create persistent journald directory
+  - name: centos-init | Install locate
+    become: yes
+    yum:
+      name: mlocate
+      state: present
+
+  - name: centos-init | Update locate db
+    command: updatedb
```

Comment:
I hadn't added previously. @mtbc : was the behavior on idr-experimental ok for you? If so, I could drop. Otherwise, the only thing that occurs to me is to setup in cron.

Reply:
I did an updatedb manually there. Waiting for `crond` to run `/etc/cron.daily/mlocate` would be fine.

Reply:
mlocate's significantly better than predecessors in my limited experience.